src/components/Tree/Tree.spec.tsx

```diff
@@ -10,10 +10,18 @@ import IconGuidelines from "@foundation/Icon/Generated/IconGuidelines";
 
 type ComponentProps = {
     nodes: TreeNodeProps[];
+    onDrop?: () => void;
 };
-const Component: FC<ComponentProps> = ({ nodes }) => {
+const Component: FC<ComponentProps> = ({ nodes, onDrop }) => {
     const [selectedId, setSelectedId] = useState<string | null>();
-    return <Tree nodes={nodes} activeNodeId={selectedId} onSelect={(id: string | null) => setSelectedId(id)} />;
+    return (
+        <Tree
+            nodes={nodes}
+            activeNodeId={selectedId}
+            onSelect={(id: string | null) => setSelectedId(id)}
```

Comment:
use NullableString instead of string | null

Reply:
Nullable* types are not in arcade ATM. That kind of change should have a separate commit (it's not related to changes done in this PR). You can check #641 , it's all done there.